update-pdf-images.sh

```diff
@@ -4,6 +4,6 @@
 
 echo "running npm ci... this may take a while"
 
-npm ci
+npm i
```

Comment:
Is this intentional?

Reply:
Yes. This forces the reinstallation of the NPM packages otherwise on the M1 it would come back missing packages.